README.md

```diff
@@ -10,14 +10,17 @@ Updates to Bottlerocket are rolled out in [waves](https://github.com/bottlerocke
 
 ### Installation
 
-We can install the Bottlerocket update operator using the recommended configuration defined in [bottlerocket-update-operator.yaml](./bottlerocket-update-operator.yaml):
+We can install the Bottlerocket update operator using the recommended configuration defined in [bottlerocket-update-operator.yaml](https://github.com/bottlerocket-os/bottlerocket-update-operator/blob/v0.2.0/yamlgen/deploy/bottlerocket-update-operator.yaml):
 
 ```sh
 kubectl apply -f ./bottlerocket-update-operator.yaml
```

Comment:
Just curious and question if customers ignore the recommended configuration link above, and directly run this command with latest changes of brupop. Will they still use latest version of `CRD` and meet same issue because `./bottlerocket-update-operator.yaml` still symlink to `yamlgen/deploy/bottlerocket-update-operator.yaml`?

Reply:
Unfortunately so. I think this continues to be a way to make a mistake until we merge #161, which should make the CRDs immutable. Then it won't matter so long as your CRD deployment is newer than the latest release.